packages/core/src/render3/node_manipulation.ts

```diff
@@ -1030,12 +1035,27 @@ export function applyStyling(
         (rNode as HTMLElement).style.removeProperty(prop);
       }
     } else {
+      // A value is important if it ends with `!important`. The style
+      // parser strips any semicolons at the end of the value.
+      const isImportant = typeof value === 'string' ? value.slice(-11) === ' !important' : value;
```

Comment:
Should `isImportant` be true if `value` is truthy??

```suggestion
      const isImportant = typeof value === 'string' ? value.slice(-11) === ' !important' : false;
```

Reply:
Perhaps use `endWith()` here?

Reply:
I wanted to use it, but it would require people to load another polyfill.

Reply:
Only for IE right?

Reply:
Right, although it's still something that we'd have to communicate. I just looked through the runtime code and we only have one place where `endsWith` is used and it's behind an `ngDevMode` check.

Reply:
What about this one: https://github.com/angular/angular/blob/d1ea1f4c7f3358b730b0d94e65b00bc28cae279c/packages/common/src/location/util.ts#L27

Reply:
That's a good find, I was only looking at Ivy-specific code. I'll switch it to `endsWith` when I'm addressing the rest of the feedback.